package.json

```diff
@@ -55,6 +55,7 @@
     "babel-preset-es2015": "^6.24.1",
     "crypto-js": "^3.1.9-1",
     "flot": "https://github.com/flot/flot#v0.8.3",
+    "flot-axislabels": "https://github.com/markrcote/flot-axislabels#release-2.0.1",
```

Comment:
FYI, we have a version of this with a proper `package.json` that makes node/yarn happier: https://github.com/j-white/flot-axislabels#master

Reply:
Thanks, will use that one.